src/cpu/cgroup.rs

```diff
@@ -79,8 +79,9 @@ pub fn read() -> Result<CgroupCpuMeasurement> {
     use super::cgroup_v2::read_and_parse_v2_sys_stat;
 
     let v2_sys_fs_file = Path::new("/sys/fs/cgroup/cpu.stat");
+    let v2_sys_fs_cpu_max_file = Path::new("/sys/fs/cgroup/cpu.max");
     if v2_sys_fs_file.exists() {
```

Comment:
```suggestion
    if v2_sys_fs_file.exists() {
      let v2_sys_fs_cpu_max_file = Path::new("/sys/fs/cgroup/cpu.max");
```
No need to set the variable if the cgroups v2 CPU file doesn't exist.